MaterialLib/MPL/Properties/CapillaryPressureSaturation/SaturationVanGenuchten.h

```diff
@@ -15,20 +15,35 @@ namespace MaterialPropertyLib
 class Medium;
 class Phase;
 class Component;
-
-/// The van Genuchten soil characteristics function.
-///
-/// This property must be a medium property, it computes the saturation of the
-/// wetting phase as function of capillary pressure.
+/**
+ *   \brief van Genuchten water retention model
```

Comment:
```suggestion
 *   \brief The van Genuchten water retention model.
```

Reply:
Not sure, if water is the only valid fluid...

Reply:
Changed it to `The van Genuchten capillary pressure model.`